modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -92,8 +104,8 @@ public override void _ExportFile(string path, string type, string[] features)
 
             if (!ProjectContainsDotNet())
             {
-                _maybeLastExportError = $"This project contains C# files but no solution file was found at the following path: {GodotSharpDirs.ProjectSlnPath}\n" +
-                    "A solution file is required for projects with C# files. Please ensure that the solution file exists in the specified location and try again.";
+                GetExportPlatform().AddMessage(EditorExportPlatform.ExportMessageType.Error, "Export .NET Project", $"This project contains C# files but no solution file was found at the following path: {GodotSharpDirs.ProjectSlnPath}\n" +
```

Comment:
Aren't we showing two error messages here, since we call `AddExceptionMessage` when catching the subsequent `InvalidOperationException`?

Reply:
No, we are not catching this exception. This is the `_ExportFile` method which is called by the engine, so we don't surround it by a try-catch (at least not here, it's catched by `CSharpInstanceBridge.Call` and that only prints it to the console).

Reply:
You're right. I got bamboozled by the GitHub mobile UI, and didn't see lines were hidden in between 🥲